### ISSUE TITLE
Ignoring configured affinity for vscodevim.vim

### ISSUE THREAD

**Issue by svipas** [VSCodeVim/Vim#8368]
**Describe the bug**

I have this setting:

```
  "extensions.experimental.affinity": {
    "vscodevim.vim": 1
  },
```

And, in developer tools → console, I get this:

```
workbench.desktop.main.js:sourcemap:620  INFO Ignoring configured affinity for 'vscodevim.vim' because the extension is unknown or cannot execute for extension host kind: LocalWebWorker.
```

- Extension (VsCodeVim) version: 1.25.2
- VSCode version: 1.77.1
- OS: macOs

**Comment by lawrence-laz** [comment]
Can you check the `Developer: Show Running Extensions` window to check if the extension host is not actually running?

![image](https://user-images.githubusercontent.com/8823448/232283436-dbba058b-512b-4b3c-94fa-5ac750bab3d7.png)

I am getting the same INFO message multiple times, but it's preceded by this:
```
log.ts:421  INFO Ignoring configured affinity for 'vscodevim.vim' because the extension is unknown or cannot execute for extension host kind: LocalWebWorker.
```
But it seems like the extension is running in a separate process as seen in `Developer: Show Running Extensions` window and the performance improved in my case.

**Comment by svipas** [comment]
Yeah, you're right I see that it is in `local process 2`, but still I think it would makes sense to fix it, because it confuses and I thought that it is running in the same process.
